### PR TITLE
[Do not merge] Update Android SDK Packages in setup scripts

### PR DESCRIPTION
**Pull request by vchimev**
Updated and verified for macOS.
Updated but NOT verified for Windows - @etabakov, would appreciate your help here.

### CODE REVIEW

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
Updated and verified for macOS.
Updated but NOT verified for Windows - @etabakov, would appreciate your help here.

File: setup/native-script.ps1 [hunk 1]
```diff
@@ -97,28 +97,21 @@ if (!$env:JAVA_HOME) {
...
-# the android tool will introduce a --accept-license option in subsequent releases
-$androidExecutable = [io.path]::combine($env:ANDROID_HOME, "tools", "android")
-echo y | cmd /c "$androidExecutable" update sdk --filter "platform-tools" --all --no-ui
-echo y | cmd /c "$androidExecutable" update sdk --filter "tools" --all --no-ui
-echo y | cmd /c "$androidExecutable" update sdk --filter "android-23" --all --no-ui
-echo y | cmd /c "$androidExecutable" update sdk --filter "build-tools-25.0.2" --all --no-ui
-echo y | cmd /c "$androidExecutable" update sdk --filter "build-tools-23.0.3" --all --no-ui
-echo y | cmd /c "$androidExecutable" update sdk --filter "extra-android-m2repository" --all --no-ui
+$androidExecutable = [io.path]::combine($env:ANDROID_HOME, "tools", "bin", "sdkmanager")
+echo y | cmd /c "$androidExecutable" "platform-tools"
```
Comment (vchimev) [inline setup/native-script.ps1]: Unable to test, perhaps it should be escaped.

File: setup/native-script.ps1 [hunk 2]
```diff
@@ -97,28 +97,21 @@ if (!$env:JAVA_HOME) {
...
 		$haxmSilentInstaller = [io.path]::combine($env:ANDROID_HOME, "extras", "intel", "Hardware_Accelerated_Execution_Manager", "silent_install.bat")
 		cmd /c "$haxmSilentInstaller"
-
-		echo y | cmd /c "$androidExecutable" update sdk --filter sys-img-x86-android-23 --all --no-ui
-		echo no | cmd /c "$androidExecutable" create avd -n Emulator-Api23-Default -t android-23 --abi default/x86 -c 12M -f
-	} else {
-		echo y | cmd /c "$androidExecutable" update sdk --filter sys-img-armeabi-v7a-android-23 --all --no-ui
-		echo no | cmd /c "$androidExecutable" create avd -n Emulator-Api23-Default -t android-23 --abi default/armeabi-v7a -c 12M -f
 	}
+	echo y | cmd /c "$androidExecutable" "system-images;android-25;google_apis;x86"
```
Comment (vchimev) [inline setup/native-script.ps1]: Let's both scripts for macOS and Windows do the same.

Comment (dtopuzov) [inline setup/native-script.ps1]: Can we install `android-23` images?
Reason:
- It is the most used version
- Boots faster

Comment (vchimev) [inline setup/native-script.ps1]: Done in https://github.com/NativeScript/nativescript-cli/pull/2857.

**Comment by vchimev** [comment]
I'm closing this pull request in favour of:
https://github.com/NativeScript/nativescript-cli/pull/2857
and
...

**Pull request closed without merging**